api-compact.yaml

```diff
@@ -85,9 +91,11 @@ x-amazon-apigateway-integration: &apigateway_integration
   type: "http"
   requestParameters:
     integration.request.header.Authorization: "method.request.header.Authorization"
+    integration.request.header.Host: "${stageVariables.enterprise_host}"
```

Comment:
Yes, it may rewrite the path TO the origin, but does it rewrite the path FROM the origin (as in, the pagination "next" URI)? How does the enterprise service know what the correct pagination URIs are?

That said, I recommend filing an AWS support ticket to ask if there's a better way to pass API Gateway custom domain information to the origin, or if we really do have to hard-code a stage variable. It would be really nice to have this "just work" across any deployment of the gateway.